docs/modules/ROOT/pages/guides/v3-migration/index.adoc

```diff
@@ -0,0 +1,159 @@
+----
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+const server = new ApolloServer({
+    schema: neoSchema.schema,
+});
+----
+
+Now you'll need to do the following:
+
+[source, JavaScript, indent=0]
+----
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+const schema = await neoSchema.getSchema();
+const server = new ApolloServer({
+    schema: schema
+});
+----
+
+== Relationship changes
+This release contains an overhaul of our relationship validations, this will require a few changes to the schema.
+
+=== Many to * relationships
+To improve consistency and validation, **all** "many to *" relationships need to be defined as _required_ in the schema:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    actors: [Actor!]! @relationship(type: "ACTED_IN", direction: IN)
+    director: Director @relationship(type: "DIRECTED", direction: IN)
+}
+----
+
+Note that any other notation, such as `[Actor]` or `[Actor!]` will **not** be valid. "One to one" relationships
+such as `Director` remain unchanged.
+
+=== Relationship cardinality
+Runtime checks for relationships number of connections have been rolled in with this release, this means that some
+databases with inconsistent relationships between the schema definition and the actual data may now fail in some queries.
+For those cases, please ensure that the database is following your schema definition or update the schema to reflect the
+actual existing relationships.
+
+== Count query no longer supported
+Queries using `count` at the root level are no longer supported. For example:
+[source, graphql, indent=0]
+----
+Query {
+    usersCount
+}
+----
+
+The same operation, can now be achieved with a xref::queries#_counting_using_aggregation[count aggregation] query:
+
+[source, graphql, indent=0]
+----
+Query {
+    usersAggregate {
+        count
+    }
+}
+----
+
+== Relationship filters
+`where` filters for relationship queries now explicitly state `ALL`, `NONE`, `SINGLE`, and `SOME` as part of filter name.
+
+Queries using old relationship filters, will now need to use `{relationship}_NONE`. For example:
+
+[source, graphql, indent=0]
+----
+Query {
+  movies(where: {
+    actors: {
+      name: "John"
+    }
+  }) {
+    title
+  }
+}
+----
+
+Should be:
+
+[source, graphql, indent=0]
+----
+Query {
+  movies(where: {
+    actors_SOME: {
+      name: "John"
+    }
+  }) {
+    title
+  }
+}
+----
+
+And, instead of `_NOT`, `_NONE` should be used.
+
+NOTE: Old queries will still work in this release, but are marked as `@deprecated` and will not be available in the future.
+
+== `@ignore` directive renamed to `@computed`
+To better reflect its intended usage, the `@ignore` directive is now named `@computed`, behaviour is unchanged, so you just need to
+rename this directive in your schema.
+
+== Auth plugin system
+https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-key-sets[JSON Web Key Sets] support through `jwksEndpoint` option is no longer
```

Comment:
This is more than just `jwksEndpoint`? I would suggest this should have two subsections, one for JWT users and one for JWKS users.

Reply:
Agree, I think I'll wait until auth is merged to fix this, as we will probably want to link to the updated docs to avoid duplication